packages/contracts/test/utils/combinatorial_sets.ts

```diff
@@ -0,0 +1,25 @@
+import { BigNumber } from '@0xproject/utils';
+import * as _ from 'lodash';
+
+export const BigNumbers = [];
+
+const MAX_256_BIT_NUMBER = new BigNumber(2).pow(256).minus(1);
+
+export const positiveNaturalBigNumbers = [
+    new BigNumber(0),
+    new BigNumber(1),
+    new BigNumber(2),
+    new BigNumber(2).pow(64),
+    MAX_256_BIT_NUMBER,
+    MAX_256_BIT_NUMBER.div(2).floor(),
+    MAX_256_BIT_NUMBER.sqrt().floor(),
+];
+
+export const negativeNaturalNumbers = _.map(positiveNaturalBigNumbers, n => n.neg());
+export const naturalBigNumbers = _.concat(positiveNaturalBigNumbers, negativeNaturalNumbers);
+
+function bigNumbersToNumbers(bigNumbers: BigNumber[]): number[] {
+    return _.map(bigNumbers, (x: BigNumber) => x.toNumber());
```

Comment:
This function it looses precision and we need it to be exact. The `toNumber()` should only be used for `console.log` purposes, and that should not be done in this file.

Reply:
If we want to be able to nicely display the test sets, maybe we can turn them into objects:

```
export const uin256 {
    '2²⁵⁶ - 1': new BigNumber(2).pow(256).sub(1),
    '2²⁵⁵':     new BigNumber(2).pow(255),
    ...
}
```